source/google_cloud_storage/README.md

```diff
@@ -1,3 +1,10 @@
 # google_cloud_storage
 
+```go
+import (
+  _ "github.com/golang-migrate/migrate/v4/source/google_cloud_storage"
```

Comment:
Isn't the package name `googlecloudstorage`?

https://github.com/golang-migrate/migrate/blob/master/source/google_cloud_storage/storage.go

Also, could add section headers? e.g. one section for the import and one section for the URI

Reply:
That's what threw me off as well. Import needs the folder name, not the `package` name.